src/Dfc.CourseDirectory.Core/Search/CourseSearchQuery.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Search.Documents;
+using Dfc.CourseDirectory.Core.Search.AzureSearch;
+using Dfc.CourseDirectory.Core.Search.Models;
+using Azure.Search.Documents.Models;
+
+namespace Dfc.CourseDirectory.Core.Search
+{
+    public class CourseSearchQuery : IAzureSearchQuery<Course>
+    {
+        private const string RegionBoostScoringProfileName = "region-boost";
+
+        public string CourseName { get; set; }
+
+        public IEnumerable<string> Facets { get; set; }
+
+        public IEnumerable<string> Filters { get; set; }
+
+        public string OrderBy { get; set; }
+
+        public int Size { get; set; }
+
+        public int? Skip { get; set; }
+
+        public (string SearchText, SearchOptions Options) GenerateSearchQuery()
+        {
+            var filter = string.Join(" and ", (Filters ?? Array.Empty<string>()).Append("Status eq 1"));
```

Comment:
`Enumerable.Empty<string>()`?

Reply:
Use `nameof(Course.Status)`?

Reply:
`Array.Empty<T>()` and `Enumerable.Empty<string>()` are the same other than return type: https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Linq/src/System/Linq/Enumerable.SizeOpt.cs#L10

Reply:
That doesn't surprise me, but given that you're using it in place of an `IEnumerable<string>` it seems the more appropriate choice.